pkg/scaler/prober.go

```diff
@@ -459,11 +461,41 @@ func (p *prober) scaleTo(parentContext context.Context, msg string, replicas int
 				klog.V(4).Infof("%s: skipped because desired=%d and current=%d", prefix, replicas, s.Spec.Replicas)
 				continue
 			}
+			/*
+				Check if the scaled objects has defined any delays for the operation.
+				scaleUpDelay is the delay in seconds to wait before initiating scaleUp to ensures that the resource is scaled up after allowing sufficient time for system to recover.
+				scaleDownDelay is the dealy in seconds to wait before initiaing scaleDown to ensure that the resource is scaled down after allowing its dependents room to react.
```

Comment:
Nitpick: `delay` and `initiating`

Reply:
Thanks for highlighting. I've fixed them with latest forced commit.